Cesium for Unity does not work in Unity 2022.2.0f1

It reports:

> error CS0016: Could not write to output file 'C:\Users\runneradmin\AppData\Local\Temp\09094c8b-c05b-4ac9-ba57-e1c9f8000f72\generated\Runtime\Unity.SourceGenerators\Unity.MonoScriptGenerator.MonoScriptInfoGenerator\AssemblyMonoScriptTypes.gen.cs' -- 'Could not find a part of the path 'C:\Users\runneradmin\AppData\Local\Temp\09094c8b-c05b-4ac9-ba57-e1c9f8000f72\generated\Runtime\Unity.SourceGenerators\Unity.MonoScriptGenerator.MonoScriptInfoGenerator\AssemblyMonoScriptTypes.gen.cs'.'

The problem is that our build process adds `-generatedfilesout:"C:\Users\runneradmin\AppData\Local\Temp\09094c8b-c05b-4ac9-ba57-e1c9f8000f72\generated\Runtime"` to the csc.rsp files in the Runtime and Editor directories in order to get the Reinterop-generated output and bake it into the build. But it then erroneously includes these modified csc.rsp files in the package. This is apparently harmless in Unity 2021.3, probably because there are no generated files to write in a standard build. But in 2022.2, it tries to write "something" to that location, and fails because the directory doesn't exist.

The solution is to remove these lines from csc.rsp before packaging.

--- a/Build~/Package.cs
+++ b/Build~/Package.cs
@@ -356,8 +356,12 @@ namespace Build
                 "Editor/BuildCesiumForUnity.cs.meta",
                 "Editor/ConfigureReinterop.cs",
                 "Editor/ConfigureReinterop.cs.meta",
+                "Editor/csc.rsp",
+                "Editor/csc.rsp.meta",
                 "Runtime/ConfigureReinterop.cs",
-                "Runtime/ConfigureReinterop.cs.meta"
+                "Runtime/ConfigureReinterop.cs.meta",
+                "Runtime/csc.rsp",
+                "Runtime/csc.rsp.meta"
             };
 
             foreach (string fileToDelete in filesToDelete)

